packages/babel-polyfill-udemy-website/package.json

```diff
@@ -18,7 +18,7 @@
     "yarn": "^1.3.2"
   },
   "devDependencies": {
-    "babel-eslint": "^8.2.1",
+    "babel-eslint": "^8.2.5",
```

Comment:
I need to upgrade babel-eslint so that ESLint works with the newest version of Babel.